etc/config/wazuh.conf

```
@@ -1,7 +1,7 @@
 [agent]
 server_mgmt_api_port = "55000"
 agent_comms_api_port = "27000"
-manager_ip = "localhost"
+server_address = "localhost"
```

```suggestion
server_url = "localhost"
```
Let's use this name for now and also `registration_url` for the other API.

---

Done